src/main/resources/db/migration/V1__create_tables.sql

```diff
@@ -0,0 +1,51 @@
+create table movies
```

Comment:
Yeah, there's this DDL thing (http://slick.lightbend.com/doc/3.3.0/schemas.html#data-definition-language). I used it once (https://gitlab.com/agluszak/forumeria/blob/master/src/main/scala/io/gitlab/agluszak/forumeria/repositiories/DAO.scala#L22). 

We gain: less writing, one source of truth -- so less mistakes coming from stupid typos 
We lose: maintainability, it would be hard to modify the schema of a working database. That's the reason why migrations exist